### PR TITLE
Remove lib folder

### LINKED ISSUE #77: Ignore lib folder
There is no need to push transpiled code into the repository. It should be added into `.gitignore`. Also a `prepare` script should be added into `package.json` to make sure that `lib` is regenerated before publishing the package into npm.

### PR DESCRIPTION
**Pull request by gurisko**
Fixes #77

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,6 +72,7 @@
         "test": "jest --silent",
         "test:watch": "jest --watchAll",
         "test:server": "node ./test/server.js",
+        "prepublish": "npm run build",
         "deploy": "npm test && npm run build && npm publish"
```
Comment (hg-pyun) [inline package.json:76]: This code is duplicated. `npm run build`. So deploy execute build twice. I think we don't need this command.

**Review by hg-pyun — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (gurisko) [reply package.json:76]: You are right that `npm run build` is unnecessary here since it will be executed by the `prepublish` script.

Comment (gurisko) [reply package.json]: Amended and force pushed.

Comment (hg-pyun) [reply package.json]: What is the purpose of the pre-publish?

Comment (gurisko) [reply package.json]: It runs before `publish` script [[docs](https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-operation-order)]. It makes sure that `npm publish` will always transpile the latest `lib` folder.

Comment (hg-pyun) [reply package.json]: @gurisko It is deprecated.
image

Comment (gurisko) [reply package.json]: @hg-pyun Very good point! I replaced it with `prepare` as also suggested in the screenshot you attached.

**Review by hg-pyun — APPROVED** [review]
LGTM

**Pull request merged**